docs/developer/DEVELOP.md

```diff
@@ -272,6 +272,15 @@ It's possible to perform local development outside of containers, in the venv th
 
 See `make help` for a short summary of these _make targets_.
 
+## Logging
+
+### Podman Logs
+
+You can use podman's logging subcommand along with the `--follow` (`-f`) flag to get a better idea of what is going on in the container. This is especially helpful with the container running Django/WSGI.
+
+```sh
+$ podman logs -f osidb-service
+```
```

Comment:
I am not strongly against this but I wonder whether documenting **Podman** is not beyond OSIDB documentation. There are certainly tons of potentially useful **Podman** commands but it has its own documentation. If it was some specific OSIDB procedure, it was non-intuitive, etc. then probably let us document it here.